bblfsh-dashboard/templates/deployment-with-sidecar-bblfshd-container.yaml

```diff
@@ -30,12 +30,16 @@ spec:
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         args:
         - -bblfsh-addr=localhost:9432
+        - -bblfshctl-addr=localhost:9433
```

Comment:
I used `localhost` because that's how it was done in the previous line, but I'm not sure if it should be done something more to expose that port in `bblfshd` container, to be reachable from `dashboard` container.

Reply:
It is a sidecar container, it is reachable through localhost